### DISCUSSION TITLE
Disable Giscus for one page

### DISCUSSION THREAD

**Discussion by ryanwwest** [giscus/giscus#861] [Q&A]
I have an 'about me' markdown page that I render using Hugo+[Quartz](https://github.com/jackyzha0/quartz), but Giscus appears on these pages. I know I could figure out how to disable loading in the Javascript on this page, and maybe that's a better option, but is there any way I can blacklist the particular URL from Giscus so it doesn't load?

**Comment by davidsneighbour** [comment]
That's a question for whatever Quartz is. In Hugo, you add a template for Giscus to your layouts, and it's a simple check if you are on the about me page (or even better, if not) that can decide to show the Giscus code. This is something other than what you do when initializing Giscus.

**Reply by ryanwwest** [reply]
Thanks - Quartz is a theme but isn't that relevant here (though I added a link above). Do you by chance know what the 'simple check' you are referring to in hugo layout partial syntax is? I've used it a handful of times but am not familiar with how to check whether you're on a certain URL (I assume it will start with `{{ if ... }}`).
